samples/customchecks/monitoring3rdparty/Version_3_3/3rdPartySystem/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
```

Comment:
Is lightness a desired non-functional requirement and more important than
familiarity for the users (http listener is maybe not known but webapi is,
but of course the service is only a supporting infrastructure)?